### ISSUE TITLE
CurrentActivity incompatible with .NET 2.0 Standard

### ISSUE THREAD

**Issue by hkidd88** [jamesmontemagno/PermissionsPlugin#110]
I am creating a cross platform application that needs access to the camera. I am developing using .NET 2.0 Standard, so I cannot install the CurrentActivity NuGet plugin. Because of this, I have to redirect to OpenAppSettings(), as RequestPermissionsAsync() returns Unknown instead of Granted [I am assuming because it cannot use the CurrentActivity methods the PermissionsPlugin assumes will be available]. Is there a way to use the PermissionsPlugin with .NET 2.0 Standard and have it bring up the standard Android dialog box instead of having to redirect to the device settings?

**Comment by diegorafael** [comment]
Any luck with this? Same issue here. I've updated the packages in my project and the whole solution became a mess of references related to this incompatibility =/

**Comment by hkidd88** [comment]
@diegorafael Yes - the issue I was having was with the CurrentActivity NuGet plugin. It is compatible with .NET 2.0 Standard. The Current Activity Plugin only has to be installed in the Android project. This should solve your problem.
